CHANGES.md

```diff
@@ -1,3 +1,81 @@
+## Version 0.9.0
+
+> 2017 January 3
+
+The biggest user-facing change in this release is to the behavior of the `flatMap` (and related methods) provided by `EitherOps` for the standard library's `Either` for Scala 2.10 and 2.11. These methods now match the behavior of the `flatMap` on `Either` in Scala 2.12 in that they don't require the left-hand side types to match.
+
+For example, the following would previously compile on 2.12, but not 2.10 or 2.11:
+
+```scala
+import cats.syntax.either._
+
+sealed abstract class AppError
+case object Error1 extends AppError
+case object Error2 extends AppError
+
+val either1: Either[Error1.type, String] = Right("hi")
+val either2: Either[Error2.type, String] = Right("bye")
+
+val result: Either[AppError, String] = for {
+  v1 <- either1
+  v2 <- either2
+} yield v1 + v2
+```
+
+This code now works equivalently on all supported Scala versions.
+
+Changes:
+
+ * [#1424](https://github.com/typelevel/cats/pull/1424): `NonEmptyList` and `NonEmptyVector` are now covariant
+ * [#1506](https://github.com/typelevel/cats/pull/1506): `flatMap` provided by `Either` syntax matches 2.12's `Either#flatMap`
+ * [#1466](https://github.com/typelevel/cats/pull/1466): Improved stack safety for `StateT`
+ * [#1510](https://github.com/typelevel/cats/pull/1510): `catchNonFatal` for `Future` is now asynchronous
+
+Bug fixes:
+
+ * [#1465](https://github.com/typelevel/cats/pull/1465) and [#1507](https://github.com/typelevel/cats/pull/1507): Stack safety law for `Monad#tailRecM` is less eager and doesn't throw exceptions
+
+New type class instances:
+
+ * [#1475](https://github.com/typelevel/cats/pull/1475): `Reducible` instances for `Eval` and `Id`
+ * [#1496](https://github.com/typelevel/cats/pull/1496): `Choice` instance for `Kleisli`
```

Comment:
This just moved the `Choice` instance around, because there was an implicit ambiguity when you wanted a `Category[Kleisli, α, α]`.

Should we just put this under miscellaneous as something like "Simpler creation of some `SemigroupK` and `MonoidK` instances" ?